diff --git a/Cargo.toml b/Cargo.toml
index dc3fcf8..512c172 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -6,6 +6,7 @@ publish = false
 
 [dependencies]
 axum = "0.8"
+p9s = { path = "../../../../packages/rust", features = ["axum"] }
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 sqlx = { version = "0.8", default-features = false, features = ["postgres", "runtime-tokio", "macros", "migrate"] }
